### PR TITLE
Crdcdh 1662

### PR DESCRIPTION
**Pull request by vshand11**
Implemented the API editSubmissionCollaborators.

### CODE REVIEW

File: services/submission.js [hunk 1]
```diff
@@ -863,6 +863,196 @@ class Submission {
...
+            throw new Error(ERROR.FAILED_ADD_SUBMISSION_COLLABORATOR);
+    }
+
+    /**
+     * API: removeSubmissionCollaborator
+     * @param {*} params 
+     * @param {*} context 
+     * @returns 
+     */
+    async removeSubmissionCollaborator(params, context) {
```
Comment (n2iw) [inline services/submission.js:1029]: It seems this function is no longer used?

File: services/submission.js [hunk 2]
```diff
@@ -863,6 +863,196 @@ class Submission {
...
```
Comment (n2iw) [inline services/submission.js:934]: Is this function same the one at line 1029?

File: services/submission.js (hunk 1 above)
Comment (vshand11) [reply services/submission.js]: removed

File: services/submission.js (hunk 2 above)
Comment (vshand11) [reply services/submission.js]: removed

File: services/submission.js [hunk 3]
```diff
@@ -863,6 +863,130 @@ class Submission {
...
+        const result = await this.submissionCollection.update( aSubmission);
+        if (result?.modifiedCount === 1) {
+            return aSubmission
+        }
+        else
+            throw new Error(ERROR.FAILED_ADD_SUBMISSION_COLLABORATOR);
+    }
+
+    /**
+     * API: editSubmissionCollaborators
```
Comment (n2iw) [inline services/submission.js:929]: Looks like this function is same as the one on line 872?

**Review by n2iw — CHANGES_REQUESTED** [review]

**Review by n2iw — APPROVED** [review]
LGTM

**Pull request merged**